src/pages/__snapshots__/PrintPage.test.tsx.snap

```diff
@@ -531,7 +531,7 @@ exports[`renders PrintPage with votes 1`] = `
                   Precinct Number
                 </div>
                 <strong>
-                  21
+                  precinct-21
```

Comment:
https://votingworks.slack.com/archives/CEL6D3GAD/p1556658341003100
> No good reason why the mismatch, we should standardize, but we should pick the shorter approach because more characters quickly blows up the QR code.
> So VX.21.5R  

Are we going to standardize these ids?

Reply:
Standardize only insofar as the IDs are going to come from the state systems and we don't control them. So we should be internally consistent with our tests but we can't be certain what other systems will do.